assets/sources/source_clients/walmart.py

```diff
@@ -8,12 +8,12 @@ class Walmart(BaseClient):
     source = "Walmart"
     use_status = "New"
 
-    def __init__(self, product_name, product_url, product_img_alt):
+    def __init__(self, product_name, product_url, product_img_alt, **kwargs):
```

Comment:
I see kwargs were added to each of the retailers, why do we need this?

Reply:
It has to do with adding product_type in. Essentially each product has a product_type, but the product itself is actualyl scraped in the source client. so i take the self.product_type that "lives" on the product object and pass it to the source so that the ScrapedProduct object can have the correct product type.